pkg/util/port/portlist/netstat_windows.go

```diff
@@ -0,0 +1,281 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2014-present Datadog, Inc.
+
+package portlist
```

Comment:
I found another implementation which is suing similar functionality. Perhaps we can reuse some of to minimize this file sinc we do use gopsutil package already. https://github.com/shirou/gopsutil/blob/master/net/net_windows.go. Just a thought, it is not 100% fit.